include/roboteam_ai/utilities/Dealer.h

```diff
@@ -174,8 +175,17 @@ class Dealer {
      * @param indexID vector with ID numbering
      * @param scores vector with all Scores for Robots for each role and its priority
      */
-    void distribute_remove(DealerDistribute &current, std::vector<int> &indexRoles, std::vector<int> &indexID,
+    static void distribute_remove(DealerDistribute &current, std::vector<int> &indexRoles, std::vector<int> &indexID,
                            std::vector<RoleScores> &scores);
+
+    /**
+     * Distributes the forced roles first, so that other rest of the function does not need to compute extra information
+     * @param allRobots a copy of all our robots, where in the robots will be removed
+     * @param flagMap wherein the roles will be removed
+     * @param output
+     */
```

Comment:
Why are these 3 static now?

Reply:
Oh think it was a Tidy pop-up, not required indeed